### PR TITLE
Add support for multimedia evaluation package

### PR DESCRIPTION
**Pull request by rshym**

### CODE REVIEW

**Comment by rshym** [comment]
Tested (with https://github.com/xen-troops/meta-xt-rcar/pull/7) on CCPF-SK (`h3ulcb-4x2g-ab` build machine).
Video and audio playback works fine.

File: prod-devel-rcar.yaml [hunk 1]
```diff
@@ -208,7 +209,7 @@ components:
         - [XT_GUEST_INSTALL, "%{XT_GENERIC_DOMU_TAG} %{XT_DOMA_TAG}"]
         # Machine feature 'multimedia' is used to enable (VSP in domd) and (LOSSY build option in ATF)
         - [MACHINE_FEATURES_append, " multimedia"]
-
+        - [XT_MULTIMEDIA_EVA_DIR, "%{XT_MULTIMEDIA_EVA_DIR}"]
```
Comment (lorc) [inline prod-devel-rcar.yaml:212]: Please leave that empty line

**Review by lorc — COMMENTED** [review]
WIth that minor fix:

`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Comment by rshym** [comment]
`Acked-by` is added.
Rebased, tested.
Ready to be merged.

**Pull request merged**